instagram/src/components/PostsContainer/PostContainer.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import './postContainer.scss'
+
+import Post from './Post'
+
+class PostContainer extends React.Component{
+
+  render () {
+    return (
+      <>
+      {this.props.posts.map(post => {
+        return (
+          <Post likes={post.likes} increaseLikes={this.increaseLikes} key={post.imageUrl} postItem={post}/>
+        )
```

Comment:
`increaseLikes={this.increaseLikes}`, you don't have `this.increaseLikes` on this class component.